packages/redux/src/settings/selectors/configuration.ts

```diff
@@ -0,0 +1,65 @@
+import { getEntityById } from '../../entities';
+import { getError, getIsLoading } from '../reducer/configuration';
+import type { Configuration } from '@farfetch/blackout-client';
+import type { ConfigurationState } from '../types';
+import type { StoreState } from '../../types';
+
+/**
+ * Retrieves the error status of a configuration by its code.
+ *
+ * @param state - Application state.
+ *
+ * @returns Error information (`null` if there are no errors).
+ */
+export const getConfigurationError = (
+  state: StoreState,
+  configurationCode: Configuration['code'],
+) =>
+  getError(state.settings?.configurations.configuration as ConfigurationState)[
```

Comment:
Instead, change it to this, please: 

```suggestion
  getError((state.settings as SettingsState).configurations.configuration)[
    configurationCode
  ];
```

This is to intentionally force an error if the `settings` reducer is not configured on the redux store. 
Please, check all other selectors and change them to work like this.